### ISSUE TITLE
Support of Merge Request Approvers

### ISSUE THREAD

**Issue by ChristianStornowski** [PavlikPolivka/GitLabProjects#120]
It would be nice to set merge request approvers with this plugin when creating merge requests via modal layer.

https://docs.gitlab.com/ee/user/project/merge_requests/merge_request_approvals.html

**Comment by PavlikPolivka** [comment]
This is only EE functionality right? 
I generally have issues to add EE specific features because it will not work for everybody and right now plugin does not have support for detecting if the used gitlab is EE or not.

**Comment by ChristianStornowski** [comment]
Yes. It seemed to available for ee only. https://docs.gitlab.com/ee/api/merge_request_approvals.html#approve-merge-request

Maybe you could implement a Plug-in configuration like: enable ee features :-D

**Comment by PavlikPolivka** [comment]
It could be a part of server setting, But this makes this more difficult and not something I have time for right now. I will leave this open and when I will have some time I can look into it.
